library/std/src/f32.rs

```diff
@@ -1154,4 +1154,54 @@ impl f32 {
         let x = unsafe { cmath::lgammaf_r(self, &mut signgamp) };
         (x, signgamp)
     }
+
+    /// Float addition that allows optimizations based on algebraic rules.
+    #[rustc_allow_incoherent_impl]
+    #[doc(alias = "fadd_algebraic", alias = "addAlgebraic")]
+    #[must_use = "method returns a new number and does not mutate the original value"]
+    #[unstable(feature = "float_algebraic", issue = "21690")]
```

Comment:
The issue number here should be a tracking issue (i.e. based on the tracking issue template) which is specific to the feature. #21690 is not a tracking issue.

Reply:
Filed https://github.com/rust-lang/rust/issues/136469 and updated links